### DISCUSSION TITLE
hosting prices question

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/tango-feedback#10] [Q&A]
hey, would it cost any more to host another sub-domain from tangoapp.dev? for example, you have app.tangoapp.dev and beta.tangoapp.dev, and I think that you could use something like dev.tangoapp.dev as a version for more technical users.

**Comment by yume-chan** [comment]
> would it cost any more to host another sub-domain from tangoapp.dev? 

No. IIRC Vercel's free tier supports 50 domains.

> I think that you could use something like dev.tangoapp.dev as a version for more technical users.

I have some ideas of new projects based on Tango library, but have no time to implement them.
